addons/binding/pom.xml

```diff
@@ -36,7 +37,7 @@
     <module>org.openhab.binding.meteostick</module>
     <module>org.openhab.binding.milight</module>
     <module>org.openhab.binding.netatmo</module>
-    <module>org.openhab.binding.network</module>
+   <!-- <module>org.openhab.binding.network</module>-->
```

Comment:
haha, I'm sorry... I had some problems compiling the module, so I wanted to ignore it while developing